content/docs/esc/environments/working-with-environments.md

```diff
@@ -760,4 +501,6 @@ Please confirm that this is what you'd like to do by typing `myorg/test`: myorg/
 Environment "myorg/test" has been removed!
 ```
 
+## Renaming an environment
+
 Environments cannot be renamed.
```

Comment:
I like the addition of the heading, but it's a bit of a bummer to see it, click it, and then be told what it says isn't possible. How about suggesting they create a copy of the environment and delete the old one? Same end result and wouldn't feel as much like a dead end.

Reply:
nice catch, added.